### PR TITLE
Fix primary button

### LINKED ISSUE #1354: Green and Teal primary buttons fail contrast accessibility in dark theme
**Package** lib-react-components

**Is your feature request related to a problem? Please describe.**
The contrast does not meet accessibility contrast standards of 4.5:1

**Additional context**

Green button with dark theme:
Screen Shot 2019-12-04 at 2 48 28 PM
Fails contrast test between the green and background color with 3.04

Teal button using dark theme:
Screen Shot 2019-12-04 at 2 48 18 PM
Fails contrast test between the teal and background color with 3.55

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-react-components

Closes #1354

Describe your changes:
I'm doing an audit of the components to make sure we're in good shape for publishing. I found a few issues with the `PrimaryButton`

- The labels for the teal and green options doesn't get the right color in light theme. I think it's a bug in the latest Grommet, but didn't have time to figure that out. I put in a workaround in the their respective theme files.

Screen Shot 2020-04-13 at 5 41 25 PM
Screen Shot 2020-04-13 at 5 41 33 PM

Fixed:
Screen Shot 2020-04-13 at 10 30 07 PM
Screen Shot 2020-04-13 at 10 30 12 PM

- I updated the stories so dark theme works again.
- I added the `PrimaryButton` to the exports so we can add it in the classifier or apps where appropriate toward #1362 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by srallen** [comment]
I've updated the dark theme versions of this to address the accessibility issue noted in #1354:

Screen Shot 2020-04-13 at 10 25 47 PM
Screen Shot 2020-04-13 at 10 25 57 PM
Screen Shot 2020-04-13 at 10 26 03 PM
Screen Shot 2020-04-13 at 10 26 10 PM

@beckyrother does this look ok?

File: packages/lib-react-components/src/PrimaryButton/themes/green.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ const theme = {
       const color = normalizeColor('status-ok', props.theme)
       const gradientShade = getGradientShade(color)
 
+      // This shouldn't be necessary where we're manually setting the color with our own ternary,
+      // but style collision is happening
+      // so the theme color under color.light isn't being applied
+      // needs more investigation, as it seems to be a Grommet bug
+      // const fontColor = props.theme.dark ? color : 'white'
```
Comment (wgranger) [inline packages/lib-react-components/src/PrimaryButton/themes/green.js:29]: Is this comment still necessary with `fontColor` commented out?

File: packages/lib-react-components/src/PrimaryButton/themes/teal.js [hunk 2]
```diff
@@ -22,6 +22,11 @@ const theme = {
       const color = normalizeColor('brand', props.theme)
       const gradientShade = getGradientShade(color)
 
+      // This shouldn't be necessary where we're manually setting the color with our own ternary,
+      // but style collision is happening
+      // so the theme color under color.light isn't being applied
+      // needs more investigation, as it seems to be a Grommet bug
+      const fontColor = props.theme.dark ? color : 'white'
```
Comment (wgranger) [inline packages/lib-react-components/src/PrimaryButton/themes/teal.js:29]: Same here: are these comments necessary as we aren't using `fontColor`?

File: packages/lib-react-components/src/PrimaryButton/themes/green.js (hunk 1 above)
Comment (srallen) [reply packages/lib-react-components/src/PrimaryButton/themes/green.js:29]: Oops! Yes, this is now cruft after I fixed the accessibility issue.

**Comment by srallen** [comment]
@wgranger I've removed the cruft, so this is ready for final review. Thanks!

**Pull request merged**